client/starwhale/api/_impl/data_store.py

```diff
@@ -158,21 +369,33 @@ def __str__(self) -> str:
 
 _TYPE_NAME_DICT = {str(v): v for v in _TYPE_DICT.values()}
 
-_ID_CUSTOM_TYPE_DICT: Dict[str, Type[Any]] = {}
-
-_CUSTOM_TYPE_ID_DICT: Dict[Type[Any], str] = {}
 
+def _get_type(obj: Any) -> SwType:
+    if isinstance(obj, list):
```

Comment:
`if isinstance(obj, list)` --> `if isinstance(obj, (list, tuple))` ?

How does the data store handle tuple type?

Reply:
more types will be supported in other PRs